SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SpatialGDKEditorSchemaGeneratorTest.cpp

```diff
@@ -238,50 +238,36 @@ const TSet<UClass*>& AllTestClassesSet()
 	return TestClassesSet;
 };
 
-TMap<FString, FString> ExpectedContents =
-{
-	TPair<FString, FString>
-		{
-			"SpatialTypeActor",
-			ExpectedFileContent::ASpatialTypeActor
-		},
-	TPair<FString, FString>
-		{
-			"NonSpatialTypeActor",
-			ExpectedFileContent::ANonSpatialTypeActor
-		},
-	TPair<FString, FString>
-		{
-			"SpatialTypeActorComponent",
-			ExpectedFileContent::ASpatialTypeActorComponent
-		},
-	TPair<FString, FString>
-		{
-			"SpatialTypeActorWithActorComponent",
-			ExpectedFileContent::ASpatialTypeActorWithActorComponent
-		},
-	TPair<FString, FString>
-		{
-			"SpatialTypeActorWithMultipleActorComponents",
-			ExpectedFileContent::ASpatialTypeActorWithMultipleActorComponents
-		},
-	TPair<FString, FString>
-		{
-			"SpatialTypeActorWithMultipleObjectComponents",
-			ExpectedFileContent::ASpatialTypeActorWithMultipleObjectComponents
-		}
+FString ExpectedContentsDirectory = TEXT("SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/ExpectedSchema");
```

Comment:
Can we throw a namespace around these variables

Reply:
They're already in an anonymous namespace.